lte/gateway/python/magma/enodebd/devices/experimental/cavium.py

```diff
@@ -118,7 +116,7 @@ def __init__(self, acs: EnodebAcsStateMachine, when_done: str):
         self.acs = acs
         self.done_transition = when_done
 
-    def get_msg(self, message: Any) -> AcsMsgAndTransition:
+    def get_msg(self) -> AcsMsgAndTransition:
```

Comment:
Can you revert the signature change to this function please - we did a minor refactor of the state machine interface recently